src/main/java/fr/openent/competences/helper/DateHelper.java

```diff
@@ -0,0 +1,44 @@
+package fr.openent.competences.helper;
+
+import fr.openent.competences.constants.DateFormat;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+public class DateHelper {
+    private static final Logger log = LoggerFactory.getLogger(DateHelper.class);
+
+    private DateHelper() {
+        throw new IllegalStateException("Utility class");
+    }
+
+    public static Date parse(String dateString, String format) {
+        Date date = null;
+        try {
+            date = new SimpleDateFormat(format).parse(dateString);
+        } catch (ParseException e) {
+            log.error(String.format("[Competence@%s::parseDate] Error when casting date. %s",
```

Comment:
Depending on Log4j or whatever logging you use, you can replace `String.format` by 

`log.error("Whatever {} should be put in here {}", this.getClass().getSimpleName(), e)`

Reply:
I have no choice on wich logger I can use, so at the moment, I have to continues with io.vertx.core.logging 😕